tasks/coverage/parser_test262.snap

```diff
@@ -2,12 +2,186 @@ commit: a1587416
 
 parser_test262 Summary:
 AST Parsed     : 45859/45859 (100.00%)
-Positive Passed: 45859/45859 (100.00%)
+Positive Passed: 45845/45859 (99.97%)
 Negative Passed: 3925/3929 (99.90%)
 Expect Syntax Error: "language/import/import-assertions/json-invalid.js"
 Expect Syntax Error: "language/import/import-assertions/json-named-bindings.js"
 Expect Syntax Error: "language/import/import-attributes/json-invalid.js"
 Expect Syntax Error: "language/import/import-attributes/json-named-bindings.js"
+Expect to Parse: "built-ins/Function/prototype/toString/class-expression-explicit-ctor.js"
+
+  × Identifier `A` has already been declared
+    ╭─[built-ins/Function/prototype/toString/class-expression-explicit-ctor.js:10:5]
+  9 │ 
+ 10 │ let A = /* before */class /* a */ A /* b */ extends /* c */ B /* d */ { /* e */ constructor /* f */ ( /* g */ ) /* h */ { /* i */ ; /* j */ } /* k */ m /* l */ ( /* m */ ) /* n */ { /* o */ } /* p */ }/* after */;
```

Comment:
Oh no!

Reply:
Don't worry! It has already been fixed in https://github.com/oxc-project/oxc/pull/4141

Reply:
Can we merge these 2 PRs, this is confusing :-/

Reply:
Done